testing/kuttl/e2e/exporter-standby/00--create-certs.yaml

```
@@ -0,0 +1,4 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+apply:
+- files/cluster-certs.yaml
```

```suggestion
- files/cluster-certs.yaml

```
Can one add a newline in a suggestion on GH?

---

yep! i just have to apply the suggestions and rebase locally.